connect/runtime/src/test/java/org/apache/kafka/connect/integration/MonitorableSourceConnector.java

```diff
@@ -140,7 +140,10 @@ public void start(Map<String, String> props) {
                                 Schema.STRING_SCHEMA,
                                 "value-" + taskId + "-" + seqno))
                         .collect(Collectors.toList());
+            } else {
+                log.info("Task stopped but still going: {}", stopped);
```

Comment:
nit: value of stopped doesn't help much with clarity here. Maybe something like `taskId`?

Reply:
Good point. That was actually leftover from debugging. Removed.